src/GafferCycles/IECoreCyclesPreview/CameraAlgo.cpp

```diff
@@ -104,6 +106,11 @@ ccl::Camera *convertCommon( const IECoreScene::Camera *camera, const std::string
 	ccam->nearclip = clippingPlanes.x;
 	ccam->farclip = clippingPlanes.y;
 	
+	// Invert the viewplane in Y so Gaffer's aperture offsets and overscan are applied in the correct direction
```

Comment:
Can we simplify this to the lines above?
```
	// Invert the viewplane in Y so Gaffer's aperture offsets and overscan are applied in the correct direction
	ccam->viewplane.bottom = -frustum.max.y;
	ccam->viewplane.top = -frustum.min.y;
```
And will we need to change the crop window to match?

Reply:
I started with something like that, but I've also been playing with getting overscan working and would need to apply the same flip to the overscan values when modifying the viewplane so figured it'd be cleaner to just flip it all in one place at the end. Happy to take the simpler path and revisit it if/when I put up a PR for overscan.

The crop window doesn't need to be flipped as it applies to the image, rather than the camera viewplane.